packages/ek-components/src/constants.js

```diff
@@ -1,16 +1,104 @@
+import { createTranslator } from 'kolibri.utils.i18n';
```

Comment:
That seems like an arbitrary restriction on how translators can be used. aiui the pattern for exported constants with translations is to export a translator (or some accessor function of it) with the constants. Otherwise the constants might as well be private to whatever Vue component is created to house the translator.

It might be worth pausing this discussion until the topic of ek-components depending on kolibri/kolibri-tools is solved though. If we can depend on kolibri here, then the PR doesn’t have to change.